Content.Server/Xenoarchaeology/XenoArtifacts/Effects/Systems/ThrowArtifactSystem.cs

```diff
@@ -43,6 +44,9 @@ private void OnActivated(EntityUid uid, ThrowArtifactComponent component, Artifa
         var lookup = _lookup.GetEntitiesInRange(uid, component.Range, LookupFlags.Dynamic | LookupFlags.Sundries);
         foreach (var ent in lookup)
         {
+            if (HasComp<GhostComponent>(ent))
+                continue;
+
             var tempXform = Transform(ent);
 
```

Comment:
Shouldn't this just check collisionLayer instead?

Reply:
Same with the others, would be better to just make this use the same method and not copy-paste the same code 3 times.